enterprise/web.beans/nbproject/project.properties

```diff
@@ -23,3 +23,4 @@ test.config.stable.includes=\
 **/*Test.class
 test.config.stableBTD.includes=\
     **/xdm/model/*Test.class
+test.unit.cp.extra=../../java/whitelist/external/glassfish-corba-omgapi-4.2.0.jar
```

Comment:
Dependency between modules in different clusters would better be handled via `project.xml` as that is more portable. I am not sure if that is possible/easy in this case however.

Reply:
had handled by adding new wrapper module java/libs.corba.omgapi module